src/core/client/stream/tabs/Live/LiveChatContainer.tsx

```diff
@@ -0,0 +1,703 @@
+
+  /** Tailing Scroll END */
+
+  useEffectAfterMount(() => {
+    beforeCommentsChanged();
+  }, [beforeComments, beforeCommentsChanged]);
+
+  useEffectAfterMount(() => {
+    afterCommentsChanged();
+  }, [afterComments, afterCommentsChanged]);
+
+  const subscribeToCommentEntered = useSubscription(
+    LiveCommentEnteredSubscription
+  );
+
+  useEffect(() => {
+    const disposable = subscribeToCommentEntered({
+      storyID: story.id,
+      orderBy: GQLCOMMENT_SORT.CREATED_AT_ASC,
+      storyConnectionKey: "Chat_after",
+    });
+
+    return () => {
+      disposable.dispose();
+    };
+  }, [story.id, subscribeToCommentEntered, afterHasMore]);
+
+  const onCommentVisible = useCallback(
+    async (visible: boolean, id: string, createdAt: string, cursor: string) => {
+      // Set the constant updating cursor
+      const key = `liveCursor:${storyID}:${storyURL}`;
+
+      const rawValue = await context.localStorage.getItem(key);
+      let current: CursorState | null = null;
+      if (rawValue) {
+        current = JSON.parse(rawValue);
+      }
+
+      if (current && new Date(createdAt) > new Date(current.createdAt)) {
+        await context.localStorage.setItem(
+          key,
+          JSON.stringify({
+            createdAt,
+            cursor,
+          })
+        );
+      } else if (!current) {
+        await context.localStorage.setItem(
+          key,
+          JSON.stringify({
+            createdAt,
+            cursor,
+          })
+        );
+      }
+
+      // Hide the "Jump to new comment" button if we can see its target comment
+      if (newComment && newComment.id === id) {
+        setNewComment(null);
+      }
+    },
+    [context.localStorage, newComment, storyID, storyURL]
+  );
+
+  const onShowConversation = useCallback(
+    (comment: LiveCommentConversationContainer_comment) => {
+      LiveChatOpenConversationEvent.emit(context.eventEmitter, {
+        storyID: story.id,
+        commentID: comment.id,
+      });
+
+      setFocusedComment(comment);
+      setConversationVisible(true);
+    },
+    [context.eventEmitter, story.id]
+  );
+
+  const onCloseConversation = useCallback(() => {
+    setFocusedComment(null);
+    setConversationVisible(false);
+  }, [setConversationVisible, setFocusedComment]);
+
+  const scrollToCommentTimeout = useRef<number | null>(null);
+  const scrollToComment = useCallback(() => {
+    if (!newComment) {
+      return;
+    }
+
+    const el = document.getElementById(`comment-${newComment.id}-bottom`);
+    if (el) {
+      el.scrollIntoView({ behavior: "smooth" });
+    }
+
+    setNewComment(null);
+  }, [newComment, setNewComment]);
+
+  const jumpToComment = useCallback(() => {
+    if (!newComment) {
+      return;
+    }
+
+    setCursor(newComment.cursor);
+
+    if (scrollToCommentTimeout.current) {
+      window.clearTimeout(scrollToCommentTimeout.current);
+    }
+
+    window.setTimeout(scrollToComment, 300);
```

Comment:
I think you forgot to set the timeout ref:
`scrollToCommentTimeout.current = window.clearTimeout(scrollToCommentTimeout.current);`